packages/gatsby-theme/src/components/Header/Header.js

```diff
@@ -43,7 +42,7 @@ const Header = forwardRef(({ children, shadow }, ref) => {
         boxShadow: shadow ? "default" : "",
       }}
```

Comment:
There seem to be some CSS for the skip component in the header. Maybe we should consider them moving them to the main Skip component.